src/torchcodec/samplers/_common.py

```diff
@@ -69,3 +69,15 @@ def _reshape_4d_framebatch_into_5d(
         pts_seconds=frames.pts_seconds.view(num_clips, num_frames_per_clip),
         duration_seconds=frames.duration_seconds.view(num_clips, num_frames_per_clip),
     )
+
+
+_FRAMEBATCH_RETURN_DOCS = """
+    Returns:
+        FrameBatch:
+            The sampled :term:`clips`, as a 5D :class:`~torchcodec.FrameBatch`.
+            The shape of the ``data`` field is (``num_clips``,
+            ``num_frames_per_clips``, ...) where ... is (H, W, C) or (C, H, W)
+            depending on the decoder options. The shape of the ``pts_seconds``
```

Comment:
Can we link directly to the docs for `dimension_order` in `VideoDecoder` here? If we can't do that, we can explicitly mention `dimension_order` and link to the docs for `VideoDecoder`.